src/trunk.c

```diff
@@ -8818,11 +8823,16 @@ trunk_node_print_branches(trunk_handle *spl, uint64 addr, void *arg)
    platform_log(
       log_handle,
       "------------------------------------------------------------------\n");
-   platform_log(
-      log_handle, "| node %12lu height %u\n", addr, trunk_height(&node));
+   platform_log(log_handle,
+                "| Page type: %s node addr=%lu height=%u next_addr=%lu\n",
```

Comment:
I think it's useful to see both current `addr` and `next_addr` for pages that are linked, so am printing that extra field here on the top.